sensor/kubernetes/localscanner/service_certificates_repository_test.go

```diff
@@ -208,13 +213,19 @@ func (s *serviceCertificatesRepoSecretsImplSuite) TestGetSecretDataMissingKeysSu
 	}
 }
 
-func (s *serviceCertificatesRepoSecretsImplSuite) TestEnsureCertsUnknownServiceTypeFailure() {
+func (s *serviceCertificatesRepoSecretsImplSuite) TestEnsureCertsUnknownServiceTypeIgnores() {
 	fixture := s.newFixture(certSecretsRepoFixtureConfig{})
-	s.getFirstServiceCertificate(fixture.certificates).ServiceType = anotherServiceType
+	s.getFirstServiceCertificate(fixture.certificates).ServiceType = unknownServiceType
+	ctx := context.Background()
+	clientSet := fake.NewSimpleClientset(sensorDeployment)
```

Comment:
Where are we using `clientSet` within the test?

Reply:
it's used for `secretsClient` which helps to check that secret for unknown service type is not persisted

Reply:
Well, but do we store anything in `secretsClient`? In l227 we are trying to `Get` something from it.